### PR TITLE
Clean up next release notes.

### PR DESCRIPTION
**Pull request by polina-c**
RELEASE_NOTE_EXCEPTION=[not user facing]

### CODE REVIEW

File: packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 This is draft for future release notes, that are going to land on
 [the Flutter website](https://docs.flutter.dev/development/tools/devtools/release-notes).
 
-# DevTools 2.22.0 release notes
+# DevTools 2.23.0 release notes
```
Comment (CoderDake) [inline packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md:4]: when you run `dart update_version.dart auto -t minor`
The release notes will automatically be cleaned up for you.
I believe the next release notes helper also does this automatically

Comment (polina-c) [reply packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md:4]: Do you suggest to remove this header or cancel this chage?

Comment (CoderDake) [reply packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md:4]: I think this should be fine actually, but we may not want to push this until our version is set to 2.23.0 as well :)

**Review by CoderDake — APPROVED** [review]

File: packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md (hunk 1 above)
Comment (polina-c) [reply packages/devtools_app/release_notes/NEXT_RELEASE_NOTES.md:4]: May be we should delete the header at all. In some next PR.

**Pull request merged**